clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/ops/loadbalancer/UpsertKubernetesLoadBalancerAtomicOperation.groovy

```diff
@@ -63,6 +63,10 @@ class UpsertKubernetesLoadBalancerAtomicOperation implements AtomicOperation<Map
 
     serviceBuilder = serviceBuilder.withNewMetadata().withName(name)
 
+    def labels = existingService.metadata?.labels ?: [:]
```

Comment:
`existingService` may be null here. Switch to `existingService?.metadata?.labels`